### ISSUE TITLE
HEVC works in Edge 91 with HEVC extensions.

### ISSUE THREAD

**Issue by nathanielcwm** [Fyrd/caniuse#5959]
Tested with this test video: https://test-videos.co.uk/vids/bigbuckbunny/mp4/h265/1080/Big_Buck_Bunny_1080_10s_1MB.mp4

HEVC seems to work and display with no issue in Edge 91.0.864.59 with HEVC extensions installed.

I'm specifically using the oem version of the extension which can be found here: https://www.microsoft.com/store/productId/9N4WGH0Z6VHQ

**Comment by sheik124** [comment]
Can confirm that Edge (Chromium) has had functional HEVC playback for quite some time now. Confirmed on both Edge 95.0.1020.53 and Edge *Dev* 97.0.1072.8.

**HEVC Extensions from Device Manufacturer** is installed, but there is also a long since unfixed *regression* in Microsoft's extension that breaks playback of HEVC encoded with B-frames and causes stuttering. I forget if it's greater than a certain # of b-frames, or just using them at all. There's a Technet discussion or Edge "issue" for it somewhere. Your test file doesn't seem to use any, so it works fine.

![image](https://user-images.githubusercontent.com/2312934/142778348-ed57e3b1-8719-42bd-bdc3-6c1395603c8a.png)


Both your test suite HEVC video and even this .mkv `https://dl5.webmfiles.org/video-h265.mkv` play just fine, albeit the latter only works when embedded because of the MKV container.

<div>
	<p>Github probably won't allow embedding this, and if they do, CORS will likely block it! But try it <a href="https://h265.webmfiles.org/embed-h265-video.php" target="_blank">at the source</a>, or with your own files, it will work.</p>
	<video width="400" height="250" controls>
	  <source src="https://dl5.webmfiles.org/video-h265.mkv" xtype="video/hevc; codecs="hevc, aac"">
	</video>
</div>

~~~HTML
<div>
	<p>Github probably won't allow embedding this, and if they do, CORS will likely block it! But try it <a href="https://h265.webmfiles.org/embed-h265-video.php" target="_blank">at the source</a>, or with your own files, it will work.</p>
	<video width="400" height="250" controls>
	  <source src="https://dl5.webmfiles.org/video-h265.mkv" xtype="video/hevc; codecs="hevc, aac"">
	</video>
</div>
~~~

**Comment by KexyBiscuit** [comment]
After I have HEVC Video Extensions purchased and installed, edge://gpu reports the following, confirms HEVC support.
```
Video Acceleration Information
Decoding (VideoDecoder)  
Decode h264 baseline  64x64 to 4096x4096 pixels
Decode h264 main  64x64 to 4096x4096 pixels
Decode h264 high  64x64 to 4096x4096 pixels
Decode vp9 profile0  64x64 to 8192x8192 pixels
Decode hevc main  48x48 to 8192x8192 pixels
Decode hevc main 10  48x48 to 8192x8192 pixels
Decode hevc main still-picture  48x48 to 8192x8192 pixels
Decoding (Legacy VideoDecodeAccelerator)  
Decode h264 baseline  64x64 to 4096x4096 pixels
Decode h264 main  64x64 to 4096x4096 pixels
Decode h264 high  64x64 to 4096x4096 pixels
Decode vp9 profile0  64x64 to 8192x8192 pixels
Decode hevc main  48x48 to 8192x8192 pixels
Decode hevc main 10  48x48 to 8192x8192 pixels
Decode hevc main still-picture  48x48 to 8192x8192 pixels
Encoding
```

**Comment by KexyBiscuit** [comment]
Will be closed by #6271.

**Comment by Schweinepriester** [comment]
#6271 has been closed, can this issue be closed? :)
